util/test/tests/GL/GL_PixelHistory_Formats.py

```diff
@@ -0,0 +1,77 @@
+import renderdoc as rd
+import rdtest
+from typing import List
+
+def event_id(x): return x.eventId
+
+class GL_PixelHistory_Formats(rdtest.TestCase):
+    demos_test_name = 'GL_PixelHistory_Formats'
+    demos_frame_cap = 10
+
+    def check_capture(self):
+        action = self.get_first_action()
+
+        while True:
+            glclear: rd.ActionDescription = self.find_action('glClear', action.eventId)
+            action: rd.ActionDescription = self.find_action('glDraw', action.eventId+1)
+
+            if action is None:
+                break
+
+            rdtest.log.success(f'Test {action.eventId} started.')
+            self.controller.SetFrameEvent(action.eventId, True)
+
+            pipe: rd.PipeState = self.controller.GetPipelineState()
+            rt: rd.BoundResource = pipe.GetOutputTargets()[0]
+            tex: rd.ResourceId = rt.resourceId
+            sub = rd.Subresource()
+            textures: List[rd.TextureDescription] = self.controller.GetTextures()
+            def predicate(texDesc):
+                return texDesc.resourceId == tex
+            texDescription = next(filter(predicate, textures))
```

Comment:
There's a helper in `TestCase` which does this for you, `texDescription = get_texture(tex)`